app/views/evaluations/_phases_table.html.erb

```diff
@@ -14,26 +13,24 @@
         <th data-label="Challenge Title" scope="row" class="text-top">
           <%= challenge_phase_title(phase.challenge, phase) %>
         </th>
-        <td data-label="Assigned to me" class="text-top">
-          <%= assigned_submissions_count(current_user, phase.challenge, phase) %>
-        </td>
-        <td data-label="Remaining to evaluate" class="text-top">
-          <%= remaining_evaluations_count(current_user, phase.challenge, phase) %>
+        <td data-label="# of Submissions to Evaluate" class="text-top">
+          <%= link_to submissions_evaluation_path(phase) do %>
+            <%= remaining_evaluations_count(current_user, phase.challenge, phase) %> of 
+            <%= assigned_submissions_count(current_user, phase.challenge, phase) %> Submissions
+          <% end %>
         </td>
-        <td data-label="Due by" class="text-top">
+        <td data-label="Evaluation Due Date" class="text-top">
           <% if phase.evaluation_form %>
-            <%= phase.evaluation_form.closing_date %>
+            <%= phase.evaluation_form.closing_date.strftime('%m/%d/%Y') %>
           <% else %>
             N/A
           <% end %>
         </td>
-        <td>
-          <div class="display-flex flex-column grid-row grid-gap-1 padding-bottom-2">
-            <%= link_to submissions_evaluation_path(phase), class: "usa-button font-body-2xs text-no-wrap width-full" do %>
-              View Submissions
-            <% end %>
-          </div>
-        </td>  
+        <td data-label="Evaluation Status" class="text-top">
+          <span class="text-top text-no-wrap usa-tag <%= evaluation_submission_assignment_status_color(phase) %>">
```

Comment:
the status_color should reflect that same value, not the status_color for the overall phase. we either need a new function for the evaluator's status in the phase, or update the `evaluation_submission_assignment_status_color` helper function to receive the `evaluation_status` atom instead of an object with that method defined. I think I prefer the second option since the function is just mapping the value to a css class.